src/Abp.AspNetCore/AspNetCore/Mvc/Results/Wrapping/AbpActionResultWrapperFactory.cs

```diff
@@ -26,5 +26,27 @@ public IAbpActionResultWrapper CreateFor(ResultExecutingContext actionResult)
 
             return new NullAbpActionResultWrapper();
         }
+
+        public IAbpActionResultWrapper CreateFor(PageHandlerExecutedContext actionResult)
+        {
+            Check.NotNull(actionResult, nameof(actionResult));
+
+            if (actionResult.Result is ObjectResult)
```

Comment:
same here, can extract the if else logic into the a method that can be reused for `ResultExecutingContext` and `PageHandlerExecutedContext`